samples/hcls_2015.ttl

```diff
@@ -94,14 +95,14 @@ If ChEMBL is incorporated into other works, we ask that the ChEMBL IDs are prese
     idot:preferredPrefix "chembl" ;
     idot:alternatePrefix "chembldb" ;
 #Provenance and Change
-    pav:version "17"^^xsd:string, "17.0" ;
+    pav:version "17"^^xsd:string ; ###HAD TO REMOVE THIS , "17.0" ;
     dct:isVersionOf :chembl ;
     pav:previousVersion :chembl16 ;
     dct:source :pubchem-bioassay-09-01-2014 ;
     pav:retrievedFrom :pubchem-bioassay-09-01-2014 ;
     prov:wasDerivedFrom :pubchem-bioassay-09-01-2014 ;
 #Availability/Distributions
-    dcat:distribution :chembl17db, :chembl17rdf ;
+    dcat:distribution :chembl17db; ### HAD TO REMOVE THIS, :chembl17rdf ;
```

Comment:
I don't agree with this change. A version should have multiple distributions. It should work though, the ShEx seems correct as well:

 (dcat:distribution IRI)*,

It might need to be (dcat:distribution IRI*) or  (dcat:distribution IRI*)*. I'm not too sure how cardinality works in these cases.